src/net_processing.h

```diff
@@ -42,13 +42,18 @@ class PeerLogicValidation : public CValidationInterface, public NetEventsInterfa
 public:
     explicit PeerLogicValidation(CConnman* connman, CScheduler &scheduler);
 
+    /** Evict orphan txn pool entries (EraseOrphanTx) based on a newly connected block */
```

Comment:
Right now the formalations here confuse me a bit, because it's a function called `BlockConnected` which does something completely different.
I think it'd make sense to mention in these comment which interface is implemented (e.g. CValidationInterface block connected), along with what the specific implementation does (which you added).